ui/languages

```diff
@@ -5537,6 +5561,8 @@ oru	Ormuri
 orv	Old Russian
 orw	Oro Win
 orx	Oro
+# Orya
```

Comment:
Why have Latin comments for a handful of these entries an not others? Can we just drop the English comments, this is a locale list, the codes should identify them.

Reply:
I have no opinion about this. The old entries had comments with the english name, but not the 3 letters entries. I added the English comments when it was in cyrillic.